lib/diskutil.js

```diff
@@ -71,12 +71,16 @@ const run = (cmd, argv, callback) => {
 
 const asyncMap = (items, iter, callback) => {
   const results = [];
+  const errors = [];
   const next = (error, result) => {
     if (result !== undefined) {
       results.push(result);
     }
-    if (error || !items.length) {
-      callback(error, results);
+    if (error !== undefined) {
+      errors.push(error || null);
```

Comment:
Do we need `|| null` here given the if-check?

Reply:
Oh no, of course not; updated the logic to be simpler, and more robust now